core/arch/arm/include/mm/mobj.h

```diff
@@ -39,6 +39,7 @@
 struct mobj {
 	const struct mobj_ops *ops;
 	size_t size;
+	size_t granule;
```

Comment:
Currently it is used only in `set_pg_region()` to make it map regions page-by-page. This is crucial for registered buffers. It means something like "longest physically contiguous part of the mobj".   
So, in case of registered mobj, `granule` will be `SMALL_PAGE_SIZE`.  Probably, this is not the best solution. But, I have no idea how to do this better. So any suggestions will be appreciated :)